src/YoutubeIframe.js

```diff
@@ -126,6 +126,40 @@ const YoutubeIframe = (props, ref) => {
     ].forEach(webViewRef.current.injectJavaScript);
   }, [play, playerReady, mute, volume, playbackRate]);
 
+  useEffect(() => {
+    if (playerReady < 1) {
+      // no instance of player is ready
+      return;
+    }
+
+    webViewRef.current.injectJavaScript(
+      PLAYER_FUNCTIONS.loadVideoById(videoId, play),
+    );
+    // We do not need `play` prop because we should not
+    // recall the load function when the prop changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
we can maintain the current active video ID, and can skip the js injection when only play was toggled